llama/runner/requirements.go

```diff
@@ -3,16 +3,18 @@ package main
 import (
 	"encoding/json"
 	"os"
+	"strings"
 
 	"github.com/ollama/ollama/llama"
 	"github.com/ollama/ollama/version"
 )
 
 func printRequirements(fp *os.File) {
```

Comment:
I still feel like having to run this with `--requirements` should be something should avoid if we can, it could add significant latency and is more moving parts

Reply:
To add: I think we _could_ continue to use the filename approach, with the limitation that each runner would have to be strictly more "featureful" than the current ones... e.g. cpu < cpu_avx < cpu_avx2 < cpu_avx512

Reply:
I've removed all the requirements logic for now